test/unit_test/model/test_pytorch_model.py

```diff
@@ -0,0 +1,53 @@
+import unittest
+
+import mlflow
```

Comment:
You may need to update the requirements list to add mlflow package in our pipeline. If mlflow is not installed in our pipeline agents, it may fail.

Reply:
`test\requirements-test.txt`

Reply:
added into requirements file